insights/parsers/dracut_modueles.py

```diff
@@ -0,0 +1,24 @@
+"""
+Dracut modules
```

Comment:
I don't think this title is enough for this module since it's not the module itself but the configuration of the related services.

Please refer to https://github.com/RedHatInsights/insights-core/wiki#checklist-for-reviewers